### ISSUE TITLE
hello, is it possible to add more algorithms, if so, how

### ISSUE THREAD

**Issue by david739** [coinfoundry/miningcore#1616]
hello, is it possible to add more algorithms, if so, how

**Comment by xiaolin1579** [comment]
Yes, it can, but how to do it, let's look at some examples from these commits.

Yescrypt
https://github.com/xiaolin1579/miningcore/commit/682585f1d63c47af893f161ab54ea82a6dfe6bea

X16rt
https://github.com/xiaolin1579/miningcore/commit/928c8cc0c62c64fc225ff177b381c290ad7be332

**Comment by david739** [comment]
Hi, thanks for the info, I'll check it out

**Comment by says112** [comment]
ocv2

**Comment by RulezZzOr** [comment]
is someone able to add kheavyhash? is it similar like heavy hash?

**Comment by david739** [comment]
Hello, it's quite a lot of work, if anyone has a modification for any algorithm, please post it here or send it to dejv739@gmail.com, thank you
